citadel-app/src/pages/instance/overview.jsx

```diff
@@ -34,7 +34,11 @@ function InstanceOverview() {
     `/instances/${name}/stop`,
     'instance',
     'POST',
-    null,
+    () => {
+      return {
+        drone: instance.drone,
+      };
+    },
```

Comment:
This should not be necessary, name of the instance should be unique for each users